WPMedia/fusion-news-theme-blocks
# TMEDIA-906: Additional content source updates for resizer

## Description

Converting content sources to use the new resizer.

## Jira Ticket

- [TMEDIA-906](https://arcpublishing.atlassian.net/browse/TMEDIA-906)

## Acceptance Criteria

Update the following content sources to use fetch with partial caching:
- alert-bar-content-source-block
- author-content-source-block
- site-hierarchy-content-block
- tags-content-source-block

## Test Steps

1. Checkout this branch `git checkout tmedia-906`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/alert-bar-content-source-block,@wpmedia/author-content-source-block,@wpmedia/site-hierarchy-content-block,@wpmedia/tags-content-source-block`
3. http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=alert-bar-collections. (the alert_bar collection doesn't really have any images... but this should still work)
4. http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=author-api&config={%22slug%22:%22tariq-alshwaiki%22} (author images are not resized at this time, but should still work)
5. http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=site-service-hierarchy&config={%22sectionId%22:%22/news%22} (again, not really image dependent...)
6. http://localhost/pagebuilder/tools/debugger?_website=daily-intelligencer&type=content-debugger&source=tags-api&config={%22slug%22:%22test-ta%22} (no images here, but should work with published story tags)

## Author Checklist
        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
